sql-bridge/src/ast.rs

```diff
@@ -630,18 +655,22 @@ impl TryFrom<&Expr> for Selection {
             } => {
                 let ident = match expr.as_ref().try_into()? {
                     Selection::Ident(ident) => ident,
-                    _ => Err("unsupported conversion to Selection, InList currently supports only Identifiers".to_string())?,
+                    selection => Err(Error::Selection {
+                        selection: Box::new(selection.clone()),
+                        r#where: None,
```

Comment:
Seems that location is still `InList` here:
```suggestion
                        r#where: Some("InList"),
```